www/src/app/components/report/UserReport.js

```diff
@@ -22,17 +23,17 @@ export default function UserReport(props) {
 
         <h3 className="tile__subtitle">Goals for this week</h3>
         <ul className="tile__updates">
-          {props.goals.map(up => <Update key={up._id}>{up.text}</Update>)}
+          {props.goals.map(up => <Update key={up._id}>{linkify(up.text)}</Update>)}
```

Comment:
Linkification seems to be a feature of the `Update` component -- maybe move it there?